lib/SyTest/Identity/Server.pm

```diff
@@ -42,6 +47,14 @@ sub _init
    return $self->SUPER::_init( @_ );
 }
 
+=head2
```

Comment:
```suggestion
=head2 rotate_keys
```

likewise for a bunch of the others.

btw in case you weren't aware: you need `perldoc lib/SyTest/Identity/Server.pm` to see the output

Reply:
I was not, thank you!